DotNet/README.md

```diff
@@ -37,6 +37,64 @@ http://[yourmachine]/DotNet/proxy.ashx?http://services.arcgisonline.com/ArcGIS/r
 ```
 * Security tip: By default, the proxy.config allows any referrer. To lock this down, replace the  ```*``` in the ```allowedReferers``` property with your own application URLs.
 
+## .NET Proxy Additional Configuration Settings
+
+The .NET proxy supports specifying an alternative TraceListener implementation for logging. This enables other logging methods such as logging to a database or enterprise logging. You can specify the alternative TraceListener in the proxy config file.
+
+* **logMethod=["class"|"file"|""]** : When set to "class" the log will look at the `logClassTypeName` attribute for the name of a class to use for logging. When set to "file" or if omitted, then the logger will log to a file if such file name is specified with the `logFile` attribute.
```

Comment:
What's the default logMethod? I'm assuming "file"...

Reply:
"file". I'll update the doc.